R/scova.R

```diff
@@ -63,41 +63,13 @@ scova <- R6::R6Class(
       # Identify columns with no variance and remove them
       variance_per_column <- apply(mm, 2, var)
       relevant_columns <- which(variance_per_column != 0)
-      mm_reduced <- mm[, relevant_columns]
+      mm_reduced <- mm[, relevant_columns, drop = FALSE]
```

Comment:
just in case this becomes a 1 column matrix, stop it from being converted to a vector with colnames removed